cmd/cp.go

```diff
@@ -119,12 +118,14 @@ func walkDirFunc(path string, d fs.DirEntry, err error) error {
 		fmt.Println(copyLogFilePath)
 
 		responseChan = make(chan tasks.BackupFileResponse, copyQueueLen)
+		var fileID uint = 0
 		go service.HandleFilesCopyResponse(copyLogFile, responseChan)
-		service.RequestFilesCopy(file, responseChan)
+		service.RequestFilesCopy(file, responseChan, &fileID)
```

Comment:
Instead of adding the `fileID` parameter, which is not used further down, how about turning it into an internal variable of `service.RequestFilesCopy`